tests/all/globals.rs

```diff
@@ -145,7 +145,7 @@ fn i31ref_global_get() -> Result<()> {
             let global = Global::new(
                 &mut store,
                 GlobalType::new(ValType::I31REF, mutability),
-                val.clone(),
```

Comment:
Were these clones being removed part of a separate clippy lint that was firing? Perhaps another good candidate to enable at the workspace level? (find to do in a separate PR)

Reply:
I believe so, yes. I might be able to get to that this weekend.

Reply:
I took a look, and it seems like `Cargo.toml` already warns about `clone_on_copy`.

```
[workspace.lints.clippy]
# The default set of lints in Clippy is viewed as "too noisy" right now so
# they're all turned off by default. Selective lints are then enabled below as
# necessary.
all = { level = 'allow', priority = -1 }
clone_on_copy = 'warn'
map_clone = 'warn'
uninlined_format_args = 'warn'
unnecessary_to_owned = 'warn'
manual_strip = 'warn'
```